docscan/session/create/check/liveness_test.go

```diff
@@ -25,6 +25,7 @@ func ExampleRequestedLivenessCheckBuilder() {
 
 func TestRequestedLivenessCheckBuilder_MaxRetriesIsOmittedIfNotSet(t *testing.T) {
 	check, err := NewRequestedLivenessCheckBuilder().
+		ForZoomLiveness().
```

Comment:
Should `ForZoomLiveness()` be a separate example?

Reply:
I just added this because the liveness check isn't valid if it doesn't have a type. So we're not testing ZoomLiveness here, we're just making it a more valid example

Reply:
Sorry, I misunderstood. Fixed in bb85135